### PR TITLE
chore(docs): Update web app page to use nargo v 0.31

### PR DESCRIPTION
**Pull request by critesjosh**
# Description

Update the tiny web app tutorial to use noir v0.31.0.

## Problem\*

Resolves https://github.com/noir-lang/tiny-noirjs-app/issues/6

## Additional Context

[This PR](https://github.com/noir-lang/tiny-noirjs-app/pull/7) should be merged after merging this one. 

## Documentation\*

Check one:
- [ ] No documentation needed.
- [x] Documentation included in this PR.
- [ ] **[For Experimental Features]** Documentation to be submitted in a separate PR.

# PR Checklist\*

- [x] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: docs/docs/tutorials/noirjs_app.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ As for `Nargo`, we can follow the [Nargo guide](../getting_started/installation/
 curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
 ```
 
+Follow the instructions on [this page](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg/cpp/src/barretenberg/bb#installation) to install `bb`. 
+Version 0.41.0 is compatible with `nargo` version 0.31.0, which you can install with `bbup -v 0.46.1` once `bbup` is installed.
```
Comment (Savio-Sou) [inline docs/docs/tutorials/noirjs_app.md:34]:
```suggestion
Version 0.41.0 is compatible with `nargo` version 0.31.0, which you can install with `bbup -v 0.41.0` once `bbup` is installed.
```

File: docs/docs/tutorials/noirjs_app.md [hunk 2]
```diff
@@ -310,9 +327,7 @@ Time to celebrate, yes! But we shouldn't trust machines so blindly. Let's add th
 
 ```js
 display('logs', 'Verifying proof... ⌛');
-const verificationKey = await backend.getVerificationKey();
-const verifier = new Verifier();
-const isValid = await verifier.verifyProof(proof, verificationKey);
+const isValid = await backend.verifyProof(proof);
```
Comment (Savio-Sou) [inline docs/docs/tutorials/noirjs_app.md:330]: Did we lose the ability to cache and verify with cached verification key?

File: docs/versioned_docs/version-v0.32.0/tutorials/noirjs_app.md [hunk 3]
```diff
@@ -30,6 +30,9 @@ As for `Nargo`, we can follow the [Nargo guide](../getting_started/installation/
 curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
 ```
 
+Follow the instructions on [this page](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg/cpp/src/barretenberg/bb#installation) to install `bb`. 
+Version 0.41.0 is compatible with `nargo` version 0.31.0, which you can install with `bbup -v 0.46.1` once `bbup` is installed.
```
Comment (Savio-Sou) [inline docs/versioned_docs/version-v0.32.0/tutorials/noirjs_app.md:34]:
```suggestion
Version 0.41.0 is compatible with `nargo` version 0.31.0, which you can install with `bbup -v 0.41.0` once `bbup` is installed.
```

**Review by Savio-Sou — APPROVED** [review]
Minor comments above that aren't merge-blocking.

File: docs/docs/tutorials/noirjs_app.md (hunk 2 above)
Comment (critesjosh) [reply docs/docs/tutorials/noirjs_app.md:330]: no we did not. I'll revert it back to how it was to make it clear that it is possible.

Comment (critesjosh) [reply docs/docs/tutorials/noirjs_app.md:330]: added it as a comment

**Pull request merged**